app/controllers/carted_products_controller.rb

```diff
@@ -1,26 +1,28 @@
 class CartedProductsController < ApplicationController
-  def create 
-    carted_product = CartedProduct.new(
-    quantity: params[:quantity].to_i,
-    product_id: params[:product_id],
-    user_id: current_user.id,
-    grind_id: params[:grind_id],
-    order_id: params[:order_id], #this will most likely need updating with Stripe specific info 
-    status: "carted"
-    ) 
-    carted_product.save
-    flash[:success] = "Order Created!"
-    redirect_to "/cart"
+  def create
+    customer_id = customer_signed_in? ? current_customer.id : session['session_id'] 
```

Comment:
For now, anonymous users get assigned the `session_id` as their user_id. I'm not sure if this good...I'm open to suggestions.